### DISCUSSION TITLE
applying sharepoint template creates duplicate navigation links

### DISCUSSION THREAD

**Discussion by abutt001** [pnp/powershell#3197] [Q&A]
When I use powershell script to apply SharePoint template, it creates duplicate navigation links on the destination site?

**Comment by StreamCalm** [comment]
Sorry to necropost, but for the benefit of future Googlers...

In my setup (creating a site script from a template site, then calling a site script from Power Automate via HTTPS request) the following seems to work.

When adding links to the navigation using this format /sites/TemplateSite/DocLibName/**Forms/AllItems.aspx** duplicate links are created by the site script.

When adding links to the navigation using _this_ format /sites/TemplateSite/DocLibName/ **no** duplicates are created by the site script.

I have absolutely no idea why this is the case, so YMMV I imagine, but works for me!
